### PR TITLE
etcher: init at 1.5.60

### PR DESCRIPTION
**Pull request by Shou**
We add Etcher, the Electron image flasher.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/tools/misc/etcher/default.nix [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchurl
+, patchelf
```
Comment (cleverca22) [inline pkgs/tools/misc/etcher/default.nix]: this `patchelf` argument is not used anywhere and could be removed

Comment (Shou) [reply pkgs/tools/misc/etcher/default.nix]: Thanks, I've fixed that now

**Comment by mmahut** [comment]
Do you mind squashing your commits please?

**Comment by mmahut** [comment]
@GrahamcOfBorg build etcher

**Comment by mmahut** [comment]
@GrahamcOfBorg build etcher

**Comment by Shou** [comment]
I've removed some unused libraries (`libnode.so` and `libffmpeg.so`) that `electron_3` already provides. They take a combined ~25MB so I think it's worth getting rid off.

**Comment by juaningan** [comment]
Getting this error in a xmonad desktop environment (no gnome) when I try to open image selection dialog:

```
(electron:14866): GLib-GIO-ERROR **: 15:25:23.928: No GSettings schemas are installed on the system
Trace/breakpoint trap (core dumped)
```

**Comment by Datamats** [comment]
Adding `gtk3` and `wrapGAppsHook` as *buildInputs* should solve the file selection issue.

Also the desktop file should be changed to just exec `balena-etcher-electron` instead of `/opt/balenaEtcher/balena-etcher-electron` which does not exist.

**Comment by Shou** [comment]
@juaningan @Datamats thanks, I should've resolved this now

**Review by Myaats — APPROVED** [review]

**Review by Br1ght0ne — COMMENTED** [review]
LGTM, checked with `nix-review` on NixOS x86_64.

**Review by Br1ght0ne — APPROVED** [review]

**Comment by Shou** [comment]
Squashed commits

**Comment by mmahut** [comment]
@GrahamcOfBorg build etcher

**Review by Lassulus — APPROVED** [review]
tested with nix-review

**Pull request merged**